packages/core/core.ts

```diff
@@ -71,6 +71,17 @@ declare global {
 }
 
 export async function initFuzzing(options: Options) {
+	const isAsterisks = options.includes.indexOf("*");
+	const isNodeMods = options.excludes.indexOf("node_modules");
+	if (isAsterisks === -1 && isNodeMods !== -1) {
```

Comment:
This would also remove `node_modules` if explicitly specified, wouldn't it? A use-case could be someone wanting to only test a part if their code, but a node dependency is named similarly and would be included without the explicit `node_modules` exclude.

Reply:
How would you distinguish between the default and explicitly set in this case? Remove the default value and add a condition that if `includes` && `excludes` are both empty set the defaults?